Packs/cisco-ise/pack_metadata.json

```
@@ -2,7 +2,7 @@
   "name": "Cisco ISE",
   "description": "Next-generation secure network access.",
   "support": "xsoar",
-  "currentVersion": "1.0.0",
+  "currentVersion": "5.5.0",
```

this is the pack version, lets bump it as a minor.
```suggestion
  "currentVersion": "1.1.0",
```

---

fixed